packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -141,8 +143,7 @@ export default function VisualEditor( { styles } ) {
 			deviceType: __experimentalGetPreviewDeviceType(),
 			isWelcomeGuideVisible: isFeatureActive( 'welcomeGuide' ),
 			isTemplateMode: _isTemplateMode,
-			// Post template fetch returns a 404 on classic themes, which
-			// messes with e2e tests, so we check it's a block theme first.
```

Comment:
This comment appears to be there for the `editedPostTemplate` line below — I think it might be explaining why not to call `getEditedPostTemplate()` directly on classic themes, and why we are checking for `supportsTemplateMode` first. Shall we keep the comment around?